content/client-lib-development-guide/features.textile

```diff
@@ -1221,6 +1226,7 @@ h4. ClientOptions
 ** @(TO3g)@ @queueMessages@ boolean - defaults to true. If false, suppresses the default queueing of messages when connection states that anticipate imminent connection (connecting and disconnected). Instead, publish and presence state changes will fail immediately if not in the connected state
 ** @(TO3h)@ @echoMessages@ boolean - defaults to true. If false, suppresses messages originating from this connection being echoed back on the same connection
 ** @(TO3i)@ @recover@ string - A connection recovery string, specified with the intention of inheriting the state of an earlier connection
+** @(TO3n)@ @idempotentRestPublishing@ boolean - defaults to false. If true, "RSL1k":#RSL1k is engaged
```

Comment:
I think this should be "defaults to false for clients with version < 1.1, otherwise true".  I realise that's arguably unnecessary given this is the 1.1 spec, but I think it's useful for developers when implementing now to know that they can bundle into 1.0 so long as it defaults to false.